server/test/suites/categoryRouter.test.js

```diff
@@ -166,4 +168,104 @@ module.exports = function categorySuite() {
         });
     });
   });
+
+  describe('/getCategoryItems/:categoryId', () => {
+    before(async () => {
+      await item1.save();
+      await item2.save();
+    });
+
+    after(async () => {
+      await Item.findByIdAndDelete(item2._id);
```

Comment:
So here, we only delete one because the other is used in later tests? A comment on line 234 may be helpful.

Reply:
Actually, looking at the rest of the code, you may just want to explicitly load in what you need for each test suite and remove them all at the end to make it more clear. I don't know if this is the best solution, so I will approve and we can see what our hosts say?

Reply:
discussed offline